ldap/servers/slapd/back-ldbm/filterindex.c

```diff
@@ -1054,13 +1054,55 @@ keys2idl(
     int allidslimit)
 {
     IDList *idl = NULL;
+    Op_stat *op_stat;
+    PRBool collect_stat;
 
     slapi_log_err(SLAPI_LOG_TRACE, "keys2idl", "=> type %s indextype %s\n", type, indextype);
+
+    /* Before reading the index take the start time */
+    op_stat = op_stat_get_operation_extension(pb);
```

Comment:
Sould we get the op_stat if collect_stat is false ?

Reply:
I agree. I changed the patch